.env.local.example

```diff
@@ -1,4 +1,5 @@
 REACT_APP_LOCAL___ADMIN=
+REACT_APP_LOCAL___ADMIN_DOMAIN=<optional 'db.eu.fauna.com' or 'db.us.fauna.com'>
```

Comment:
Please move these optional values to a comment around line 37 in scripts/setup.js 

This way, anyone using the classic Region Group doesn't need to set a value here.

Line should read:
```
REACT_APP_LOCAL___ADMIN_DOMAIN=
```

Reply:
👍